coralproject/talk
# off by 1 errors

## What does this PR do?
Addresses https://vmproduct.atlassian.net/browse/CORL-2648 and https://vmproduct.atlassian.net/browse/CORL-2651


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Enable Flatten Replies
3. Create a reply chain > 7 comments long
4. Reply to a reply past the 8th level (at which point all replies should be indented at the same level) and observe that the comment is in line with the comment preceding it (as opposed to the left of it).
5. Disable Flatten Replies
6. Create another reply chain 6 comments long (1 less than the maximum)
7. In another tab, reply to the 6th reply
8. In the original tab, note that the "Read more of this conversation" button appears
